pkg/healthcheck/healthcheck.go

```diff
@@ -0,0 +1,25 @@
+package healthcheck
+
+import (
+	"context"
+
+	"google.golang.org/grpc/health/grpc_health_v1"
+)
+
+type HealthChecker struct{}
+
+func (s *HealthChecker) Check(ctx context.Context, req *grpc_health_v1.HealthCheckRequest) (*grpc_health_v1.HealthCheckResponse, error) {
+	return &grpc_health_v1.HealthCheckResponse{
+		Status: grpc_health_v1.HealthCheckResponse_SERVING,
+	}, nil
+}
+
+func (s *HealthChecker) Watch(req *grpc_health_v1.HealthCheckRequest, server grpc_health_v1.Health_WatchServer) error {
+	return server.Send(&grpc_health_v1.HealthCheckResponse{
+		Status: grpc_health_v1.HealthCheckResponse_SERVING,
+	})
+}
+
+func NewHealthChecker() *HealthChecker {
```

Comment:
Please rename this function: `healthcheck.NewHealthChecker` stutters unnecessarily. Perhaps: `healthcheck.New()` ?

Please see https://blog.golang.org/package-names#TOC_3. and https://blog.golang.org/package-names#TOC_3.

Reply:
Thanks for reviewing and fixed your comments.